lessons/Projects/project-research-paper/langs/en-us/index.adoc

```diff
@@ -39,45 +45,53 @@
 
 === Investigate
 
 [.lesson-instruction]
 - We are going to spend a lot of class time working on our research projects.
-** _It is up to you how much work students do at home versus in class. Providing work time during class is of tremendous benefit to students so that they have space to ask questions and collaborate with peers._
+** _It is up to you how much work students do at home versus in class. Providing work time during class is beneficial because it encourages students to ask questions and collaborate with peers._
 - To ensure that everyone completes the project, there are some intermittent deadlines you should know about.
-** Share with student any pertinant deadlines, e.g.: Will you check  progress intermittently? Will students submit a rough draft before the submit a final draft? Is there a deadline for the slides, and another deadline for the paper (if you are opting to do both}? Will students present their projects to the class on a certain date?_
+** _Share with student any pertinent deadlines, e.g.: Will you check progress intermittently? Will students submit a rough draft before they submit a final draft? Is there a deadline for the slides, and another deadline for the paper (if you are opting to do both)? Will students present their projects to the class on a certain date?_
+
+@teacher{Pacing of this project will vary from classroom to classroom. Some students will need more structure and deadlines from you, while others will thrive with individual freedom. Do what works best for your students - but have a clear idea of how much time will be needed for your students to meet your expectations.
 
-@teacher{We recommend printing and distributing the student-facing rubric to help students understand the scope of the project and your expectations at the outset.
+We recommend printing and distributing the student-facing rubric to help students understand the scope of the project and your expectations at the outset.
 
-The next set of lesson instructions involves a walk-through of the slide deck template, with periodic discussion the relevant rubric rows. Teachers are welcome and encouraged to edit and adapt the rubric for their unique classroom context.}
+The next set of lesson instructions involves a walk-through of the slide deck template, with interwoven discussion the relevant rubric rows. Teachers are welcome and encouraged to edit and adapt the rubric for their unique classroom context.}
 
 [.lesson-instruction]
-- To give you a sense of *what* you will be creating, let's preview the Research Project slide deck template.
-- Take out your rubric! As we move through the template, ask questions about the rubric descriptors if they are unclear or confusing to you!
+- Let's preview the @starter-file{research-project} to give you a sense of *what* you will be creating.
+- But first, take out @opt-printable-exercise{research-project-rubric.adoc, "both"} @opt-printable-exercise{research-project-rubric-2.adoc, "pages"} of your rubric.
```

Comment:
It feels like this is a great example of a time to use multi-page PDFs, unless there's an explicit reason why you _want_ them on separate pages. We generally avoid this by design, of course, so let me know if you agree and I'll show you how to do it.

Reply:
Let's talk about this at our one-on-one tomorrow. I would need to make this change in the exploration project rubric too. There is no explicit reason for the pages to be separate - I didn't know I had any other option!